src/arch/ppc64/include/arch/byteorder.h

```diff
@@ -8,6 +8,38 @@
 #define PPC_BIT(bit)		(0x8000000000000000UL >> (bit))
 #define PPC_BITMASK(bs,be)	((PPC_BIT(bs) - PPC_BIT(be)) | PPC_BIT(bs))
 
+#define PPC_PLACE(val, pos, len) \
+	__builtin_choose_expr( \
```

Comment:
This is how it looked for me, I put the message next to `^`:
```
src/arch/ppc64/include/arch/byteorder.h:17:37: error: void value not ignored as it ought to be
  __builtin_choose_expr( \
  ~~~~~~~~~~~~~~~~~~~~~~~~            
   PPC_PLACE_GOOD_ARGS(val, pos, len), \
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   PPC_PLACE_IMPL(val, pos, len), \
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~   
   /* INCORRECT ARGUMENTS DETECTED */(void)0)
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~
```
Now:
```
src/arch/ppc64/include/arch/byteorder.h:17:3: error: void value not ignored as it ought to be
  /* Incorrect arguments detected in PPC_PLACE */ __builtin_choose_expr( \
                                                  ~~~~~~~~~~~~~~~~~~~~~~~~
   PPC_PLACE_GOOD_ARGS(val, pos, len), \
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   PPC_PLACE_IMPL(val, pos, len), \
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   (void)0)
   ^~~~~~~~
```

Reply:
Output in `coreboot/coreboot-sdk:65718760fa` and using toolchain built with `make crossgcc-ppc64` matches for me for old and new versions of the code. Maybe you're using an older toolchain. We probably want the message to show up there too, so it should be at the start of the macro.

Host:
```
powerpc64-linux-gnu-gcc (coreboot toolchain v2021-05-22_b2b5781bb4) 8.3.0
```
Docker:
```
powerpc64-linux-gnu-gcc (coreboot toolchain v65718760fa 2020-02-04) 8.3.0
```

Reply:
I was using our fork with `mkimage` for Skiboot, but tested with `65718760fa` and still got only one line :man_shrugging: No need to fight with docker I guess, lets just move it to first line and it will work for both cases.